packages/shortest/src/commands/init/index.ts

```diff
@@ -0,0 +1,124 @@
+import { execSync } from "child_process";
+import { readFile, writeFile } from "node:fs/promises";
+import { join } from "path";
+import { fileURLToPath } from "url";
+import { detect, resolveCommand } from "package-manager-detector";
+import pc from "picocolors";
+import { CONFIG_FILENAME, ENV_LOCAL_FILENAME } from "../../constants";
+import { addToEnv } from "../../utils/add-to-env";
+import { addToGitignore } from "../../utils/add-to-gitignore";
+
+export default async function main() {
+  console.log(pc.blue("Setting up Shortest..."));
```

Comment:
`blue` doesn't look great on a dark terminal. We should consider improving the contrast at some point.